packages/pds/src/api/com/atproto/account.ts

```diff
@@ -35,7 +35,7 @@ export default function (server: Server) {
     const email = input.body.email.toLowerCase()
 
     // throws if not
-    ensureValidHandle(handle, config.availableUserDomains)
+    handleLib.ensureValid(handle, config.availableUserDomains)
```

Comment:
Ah, looking at the test failures, I suppose it's going to want to end-up looking something like this:
```suggestion
    try {
      handleLib.ensureValid(handle, config.availableUserDomains)
    } catch (err) {
      if (err instanceof InvalidHandleError) {
        throw InvalidRequestError(err.message, 'InvalidHandle')
      }
      throw err
    }
```

Reply:
ugh yup